show/main.py

```diff
@@ -1621,6 +1621,13 @@ def protocol(verbose):
     from .bgp_frr_v6 import bgp
     ipv6.add_command(bgp)
 
+# Import kube commands
+from . import kube
+
+# A dummy call to denote that this kube is not unused
+# Else LGTM is unable to understand the indirect use through cli.group class method
+kube.hostname()
```

Comment:
Thanks for fixing the alerts. However, I think I'd rather have the alerts than the dummy function call. Since I opened the issue regarding restructuring the files, at least the alerts serve as a reminder that this needs attention.

Other thoughts here?

Reply:
I too was thinking in the same line ... But later realized that it would not work, for the following reason.

W.r.t kube, we clearly know, that the "alert from LGTM" is incorrect. This is because, it is tough for a tool to figure out this indirect usage link. But if I leave this alert as "not fixed", anytime someone touches the config/main.py or show/main.py, this alert would appear and the author & reviewers of that PR are less likely to have any idea on this indirect usage. 

Hence in cases, where we know clearly, suppressing explicitly with comments help other devs understand.

Reply:
This code is not the first one to introduce this issue, of .py importing each other. It has been there but suppressed, by duplicating code or other means, which is in sync with your point.

I guess, the issue you filed should be good enough. Else we have to watch every PR that touches these files and remember to add a comment, on why-to-ignore.

Reply:
Yeah, I see both sides to this, as well, but I think it's best to remove unnecessary code and like you said, watch every PR that touches these files and remember to add a comment about why to ignore. This might also push us to do the refactor sooner :)

Reply:
I don't really intend to carry a baggage around. Once  we finish the job, finish it clean and move on,  unless I put the refactoring work on myself in the next few weeks. At this point, we have no idea, by when this work could get done. 

BTW, I added explicit comments on that single line of code, which nearly a no-op.

Reply:
All LGTM comments resolved w/o any hack.